internal/store/rel/store.go

```diff
@@ -0,0 +1,18 @@
+package rel
+
+import (
+	"nimona.io/pkg/object"
+)
+
+type ObjectStore interface {
+	Put(object.Object, ...Option) error
+	Get(object.Hash) (object.Object, error)
+	UpdateTTL(hash object.Hash, minutes int) error
+	Delete(object.Hash) error
+}
+
+type StreamStore interface {
+	PutRelation(parent object.Hash, child object.Hash, options ...Option) error
+	GetRelations(parent object.Hash) ([]object.Hash, error)
+	Subscribe(parent object.Hash) (chan object.Hash, error)
```

Comment:
We need a way to allow users to void a subscription. Right now if a user decides to stop listening on the channel, the publisher will block. We need a way to remove the channel from the subscriptions list.